pyiron/atomistics/structure/atoms.py

```diff
@@ -1937,6 +1937,27 @@ def get_voronoi_volume(self):
         )
         return self.analyse_ovito_voronoi_volume()
 
+    def find_mic(self, v, vectors=True):
+        """
+        Find vectors following minimum image convention (mic). In principle this
+        function does the same as ase.geometry.find_mic
+
+        Args:
+            v (list/numpy.ndarray): 3d vector or a list/array of 3d vectors
+            vectors (bool): Whether to return vectors (distances are returned
+                if False)
```

Comment:
In Soviet Russia you don't review code, code reviews YOU!

...what length are we using for line breaks? I've been using 120, where this line break is unnecessary...but I just checked PEP8 and Guido recommends 79! Most of the code base seems to use 120 (where the code base cares about this at all...). This is probably because 120 is the pycharm default. 

This is a place I can accept violating PEP8, but if that's what we decide we should do it consciously, not by accident like me.

Reply:
yes, I strictly followed the 80 character limit (79? what's the matter with that guy?). But it's true that this looks horrible, so I'm gonna rather follow your suggestion :D